src/quick-start.md

```diff
@@ -0,0 +1,295 @@
+# Quick Start
+
+The Chimera stack is made of the following components:
+
+  * An arbitrary number of `AdmissionPolicy` resources: this is how policies
+    are defined inside of Kubernetes
+  * A Deployment of Chimera `policy-server`: this component loads all the
+    policies defined by the administrators and evaluates them
+  * A Deployment of `chimera-controller`: this is the controller that monitors
+    the `AdmissionPolicy` resources and interacts with the Chimera `policy-server`
+
+## Install
+
+The Chimera stack can be deployed using a helm chart:
+
+```console
+helm repo add chimera https://chimera-kube.github.io/helm-charts/
+helm install --namespace chimera --create-namespace chimera-controller chimera/chimera-controller
+```
+
+This will install `chimera-controller` on the Kubernetes cluster in the default
+configuration and will register the `AdmissionPolicy` Custom Resource. The
+components of the Chimera stack will be deployed inside of a Kubernetes
+Namespace called `chimera`.
+
+The default configuration values should be good enough for the majority of
+deployments, all the options are documented [here](https://chimera-kube.github.io/helm-charts/#configuration).
+
+The Chimera Policy Server is completely managed by the chimera-controller.
+
+## Chimera Policies
+
+Enforcing policies is by far the most common operation a Kubernetes
+administrators will perform. You can declare as many policies as you want,
+targeting any kind of Kubernetes resource and type of operation that can be
+done against them.
+
+The `AdmissionPolicy` resource is the core of the Chimera stack: this is
```

Comment:
Is that validating or mutating ? Does Chimera support both ?

Reply:
fixed